shared/src/create_account.rs

```diff
@@ -0,0 +1,72 @@
+use crate::mint::{mint, MintParams};
+use rand::Rng;
+use starknet_crypto::{pedersen_hash, FieldElement};
+use starknet_signers::SigningKey;
+use url::Url;
+
+// Cairo string of "STARKNET_CONTRACT_ADDRESS"
+const CONTRACT_ADDRESS_PREFIX: FieldElement = FieldElement::from_mont([
+    3829237882463328880,
+    17289941567720117366,
+    8635008616843941496,
+    533439743893157637,
+]);
+
+#[derive(Debug)]
+pub struct AccountConfiguration {
+    pub class_hash: FieldElement,
+    pub salt: FieldElement,
+    pub public_key: FieldElement,
+}
+
+pub async fn create_account(base_url: &Url) {
+    let key = SigningKey::from_random();
+
+    let salt = SigningKey::from_random().secret_scalar();
```

Comment:
This suggests that salt has anything to do with private key but that is not true. Can we generate random felt without using `SigningKey` struct?